repository-data/webfiles/src/main/resources/site/freemarker/include/imports.ftl

```diff
@@ -302,4 +302,3 @@
 
     <#return { "left": leftHash, "right": rightHash } />
 </#function>
-
```

Comment:
looks like no changes to imports.ftl apart from whitespace and newline updates, so worth removing the file from the commit.